tests/testFileMaps.py

```diff
@@ -0,0 +1,111 @@
+import os
+import unittest
+import lsst.utils.tests as utilsTests
+
+from lsst.sims.utils import defaultSpecMap
+
+class FileMapTest(unittest.TestCase):
+
+
+    def verifyFile(self, file_name, dir_name):
+        """
+        Verify that specMape[file_name] results in a file in dir dir_name
```

Comment:
specMape -> specMap?  If you change this, will you clarify what this verifies? specMap[file_name] returns the correct full path to the sed file?

Also, no need to change, but DM coding standard (and my linter)  suggests spaces around plus signs.